UpdatedImageProcessing/TargetDetection/target_detection_settings.py

```diff
@@ -0,0 +1,51 @@
+import os
+from SyntheticDataset2.ImageCreator.settings import Settings
+
+class TargetDetectionSettings(object):
+    TARGET_MAPS_PATH = "/Users/zyin/Desktop/Synthetic Dataset/Answers/target_maps"
+    TARGET_MAPS_ANSWERS_PATH = "/Users/zyin/Desktop/Synthetic Dataset/Answers/target_maps_answers"
+
+    TARGET_DETECTION_REPORT_SAVE_PATH = "/Users/zyin/Desktop/Target Detection Report"
+
+    NUMBER_OF_TARGET_MAPS = sum(os.path.isfile(os.path.join("/Users/zyin/Desktop/Synthetic Dataset/Answers/target_maps", f)) for f in os.listdir("/Users/zyin/Desktop/Synthetic Dataset/Answers/target_maps"))
```

Comment:
This shouldn't require me to put in the variables that were just defined above. Please use the variables instead of a user-inputted path here.